crates/breez-sdk/core/src/sdk.rs

```diff
@@ -949,7 +943,7 @@ impl BreezSdk {
     async fn initialize_spark_private_mode(&self) -> Result<(), SdkError> {
         if !self.config.private_enabled_default {
             ObjectCacheRepository::new(self.storage.clone())
-                .save_spark_private_mode_initialized()
+                .save_spark_private_mode(false)
```

Comment:
Here you could actually be in private mode? The private mode was 'initialized', because we don't have to tell the server if we want to be private. But if we don't care if we're private or not (`private_enabled_default == false`), that doesn't mean were 'not private'.